build.gradle

```diff
@@ -15,8 +15,10 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation'org.springframework.boot:spring-boot-starter-thymeleaf'
```

Comment:
Try to maintain the style used in the file. In this case:
```
implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
developmentOnly 'org.springframework.boot:spring-boot-devtools'
```
instead of
```
implementation'org.springframework.boot:spring-boot-starter-thymeleaf'
developmentOnly("org.springframework.boot:spring-boot-devtools")
```

Reply:
Same with indentation.

Reply:
Solved